src/Fhir.Anonymizer.Core/AnonymizationConfigurations/AnonymizerConfiguration.cs

```diff
@@ -7,6 +7,9 @@ namespace Fhir.Anonymizer.Core.AnonymizerConfigurations
     [DataContract]
     public class AnonymizerConfiguration
     {
+        [DataMember(Name = "fhirPathRules")]
+        public Dictionary<string, string>[] FhirPathRules { get; set; }
+
         [DataMember(Name = "pathRules")]
```

Comment:
In the end pathRules (v1) will be removed right?

Reply:
Yes, we should remove it at end.

Reply:
do we have user story to track it?